tests/integration_tests/README.md

```diff
@@ -39,9 +39,7 @@ Please see existing batch scripts for the data specifics.
 - Finally, the script submits another job to the queue.
 
 ## Setup
-
-Tests require that a Github token is set up and included in the `.bashrc` of
-the account that runs the tests.
+Tests require that a Github token is set up and included in the `.bashrc` of the account that runs the tests. On Summit instead of adding an environment variable, change the `$GIT_TOKEN` entry with your actual token
```

Comment:
line length ;-)  
Thanks for the comment though!

Reply:
Okay! :)